src/classic/clvm/casts.rs

```diff
@@ -107,6 +107,8 @@ pub fn bigint_from_bytes(b: &Bytes, option: Option<TConvertOption>) -> Number {
     unsigned
 }
 
+// This is the version i ported from the typescript code re: chiaminejp.  It is
+// still used in some contexts, such as node_paths.
 pub fn bigint_to_bytes(v_: &Number, option: Option<TConvertOption>) -> Result<Bytes, String> {
     let v = v_.clone();
```

Comment:
This seems pointless. The reference you take as a parameter, `v_` is immutable, and making a copy to an immutable variable doesn't seem to buy you anything. You might as well just use `v_` (and rename it ti `v`)

Reply:
I've been in the habit of this in a few languages, mainly because i dislike the leaking of mutability (how it'll be used) into its declaration.  It's a style choice.
Ok will change it.

Reply:
it used to be mutable, but along the way stopped needing that.